src/main/kotlin/app/revanced/patches/youtube/layout/theme/patch/ThemePatch.kt

```diff
@@ -24,29 +22,123 @@ class ThemePatch : ResourcePatch {
         val darkThemeBackgroundColor = darkThemeBackgroundColor!!
         val lightThemeBackgroundColor = lightThemeBackgroundColor!!
 
-        context.xmlEditor["res/values/colors.xml"].use { editor ->
-            val resourcesNode = editor.file.getElementsByTagName("resources").item(0) as Element
+        val valuesPaths: List<String> = listOf(
+            "values", // Android 11 (and below)
+            "values-v31" // Android 12 (and above)
+        )
+
+        // edit the resource color file to change the theme color
```

Comment:
What do you mean by "theme color"? Please be more concise.

Reply:
What do you mean by "App colors"?

Reply:
> What do you mean by "App colors"?

I am referring to app theme, theme colors, like amoled theme. How would you define it?